packages/utils/parse.ts

```diff
@@ -129,6 +165,19 @@ export const md = (options = {}) => {
     ...options,
   })
 
+  md.use(MarkdownItGitHubAlerts, {
+    icons: {
+      note: '<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-info"><circle cx="12" cy="12" r="10"/><path d="M12 16v-4"/><path d="M12 8h.01"/></svg>',
```

Comment:
I think it should be possible to import the relevant icons from `@modrinth/assets` instead of hardcoding them here? A couple of them might need to be added to the library though

Reply:
I've added them to the modrinth/assets locally, however how would I go about using these? Markdown-it requires a string, can't take a component afaik